io/io/src/TDirectoryFile.cxx

```diff
@@ -1186,17 +1186,21 @@ TFile *TDirectoryFile::OpenFile(const char *name, Option_t *option,const char *f
 ///
 /// Returns 0 in case of error or if a sub-directory (hierarchy) with the requested
 /// name already exists.
+/// ifNotExist returns a pointer to an extant sub-directory instead 0.
```

Comment:
```suggestion
/// ifNotExist returns a pointer to an existent sub-directory instead 0.
```
typo?

Reply:
No, but I changed it to "existing", since "extant" really means "still existing" rather than "already existing"